### PR TITLE
Add App Runner Docs

### PR DESCRIPTION
**Pull request by yimipeng**

### CODE REVIEW

File: src/docs/getting-started/apprunner.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## What is App Runner?
+
+AWS App Runner is a fully managed service that makes it easy for developers to quickly deploy containerized web applications and APIs at scale and with little to no infrastructure experience. You can start with your source code or a container image, and App Runner will fully manage all infrastructure including servers, networking, and load balancing for your application. App Runner provides you with a service URL that receives HTTPS requests to your application. As an option, App Runner can also configure a continuous deployment pipeline for you.
+
+<SectionSeparator />
+
+## Getting started with X-Ray tracing for App Runner using AWS Distro for OpenTelemetry
+
+App Runner now supports sending traces to X-Ray using the AWS Distro for OpenTelemetry (ADOT). Use ADOT SDKs to collect trace data for your containerized applications, and use X-Ray to analyze and gain insights into your instrumented application. For step-by-step instruction, see [Tracing for your App Runner application with X-Ray] (http://%20https//docs.aws.amazon.com/apprunner/latest/dg/monitor-xray.html) to learn more.
```
Comment (davikeog) [inline src/docs/getting-started/apprunner.mdx:20]: "to learn more" at the end of the paragraph feels redundant. I think this sentence could be:

"For more information, see [link]."

Comment (davikeog) [inline src/docs/getting-started/apprunner.mdx:20]: Also, first mention of X-Ray in a document / topic, should be 'AWS X-Ray'.

Comment (yimipeng) [reply src/docs/getting-started/apprunner.mdx]: Thanks for the suggestion, updated. Please refer to latest commit.

Comment (yimipeng) [reply src/docs/getting-started/apprunner.mdx]: Thanks for the suggestion, updated. Please refer to latest commit.

**Comment by davikeog** [comment]
Looks good to me at this point.

**Comment by davikeog** [comment]
Looks good to me at this point.

**Review by Aneurysm9 — APPROVED** [review]

**Pull request merged**